### PR TITLE
chore(NODE-1399): Update gz references

### PR DESCRIPTION
**Pull request by andrewbattat**
NODE-1399

This PR should have no functional impact. It only updates some filenames to be agnostic to either gz or zst images, and it fixes some dummy links to be zst rather than gz.

### CODE REVIEW

**Review by Bownairo — APPROVED** [review]

File: rs/ic_os/vsock/vsock_lib/src/host/agent.rs [hunk 1]
```diff
@@ -28,7 +28,7 @@ const NODE_ID_FILE_PATH: &str = "/boot/config/node-id";
 const SETUP_HOSTNAME_FILE_PATH: &str = "/opt/ic/bin/setup-hostname.sh";
 
 // upgrade
-const UPGRADE_FILE_PATH: &str = "/tmp/upgrade.tar.gz";
+const UPGRADE_FILE_PATH: &str = "/tmp/upgrade";
```
Comment (aterga) [inline rs/ic_os/vsock/vsock_lib/src/host/agent.rs:31]: Why is there now no file extension?

File: rs/registry/admin/src/main.rs [hunk 2]
```diff
@@ -3708,7 +3708,7 @@ async fn main() {
             eprintln!("Download IC-OS .. ");
             let tmp_dir = tempfile::tempdir().unwrap().into_path();
             let mut tmp_file = tmp_dir.clone();
-            tmp_file.push("temp.gz");
+            tmp_file.push("temp-image");
```
Comment (aterga) [inline rs/registry/admin/src/main.rs:3711]: Why is there now no file extension?

File: rs/ic_os/vsock/vsock_lib/src/host/agent.rs (hunk 1 above)
Comment (andrewbattat) [reply rs/ic_os/vsock/vsock_lib/src/host/agent.rs:31]: Upgrades are agnostic to gz or zst upgrade images, so we are dropping the extension to reflect this. The upgrade works without the file extension 👍

File: rs/registry/admin/src/main.rs (hunk 2 above)
Comment (andrewbattat) [reply rs/registry/admin/src/main.rs:3711]: Similarly to above, upgrades are agnostic to gz or zst upgrade images, so we are dropping the extension to reflect this. The code still works without the file extension 👍

**Review by aterga — APPROVED** [review]

**Pull request merged**